app/models/profile.rb

```diff
@@ -51,7 +52,7 @@ def diaspora_handle
     (self.person) ? self.person.diaspora_handle : self[:diaspora_handle]
   end
 
-  def image_url(size=:thumb_large)
+  def image_url(size: :thumb_large, fallback_to_default: true)
```

Comment:
- Metrics/CyclomaticComplexity: Cyclomatic complexity for image_url is too high. [8/6]
- Metrics/PerceivedComplexity: Perceived complexity for image_url is too high. [11/7]